src/components/Home/Carousel/Carousel.js

```diff
@@ -0,0 +1,28 @@
+import React,{useEffect,useState} from 'react'
+import axios from "axios"
+import { Carousel } from "react-responsive-carousel"
```

Comment:
try to create custom one

Reply:
if you are not using react-responsive-carousel, then remove it.
remove unnecessary code

Reply:
done